static/nodejs_server/bar_tile.ts

```diff
@@ -72,7 +72,10 @@ function getBarChartSvg(
   const tileContainer = document.createElement("div");
   tileContainer.setAttribute("id", tileProp.id);
   document.getElementById(DOM_ID).appendChild(tileContainer);
-  draw(tileProp, chartData, tileContainer, SVG_WIDTH);
+  draw(tileProp, chartData, tileContainer, SVG_WIDTH, true);
+  const chartSvg = tileContainer.querySelector("svg");
+  // viewBox attribute throws off sizing in node server
```

Comment:
This is nice, because cairosvg was unable to deal with viewBox too.  Does the SVG still have style at the top-level?

Reply:
sorry, what do you mean style? if you mean height and width etc, yes, the svg will still have those things